app.js

```diff
@@ -1190,6 +1190,25 @@ define(function(require) {
 
 			monster.ui.tooltips(parent);
 
+			parent.find('#unassigned_select_all').on('click', function() {
+				var $this = $(this);
+				var $selectAllLabel = $('.unassigned_select_all_label');
+				var isActive = $this.hasClass('active');
+				var $checkboxes = parent.find('#unassigned_numbers .unassigned-number input[type="checkbox"]');
+
+				if (isActive) {
+					$selectAllLabel.html(self.i18n.active().select_all);
```

Comment:
We usually don't use this method as our HTML is stored in templates. Use `jquery#text` instead.
```suggestion
					$selectAllLabel.text(self.i18n.active().select_all);
```

Reply:
Done, thanks!